layer.yaml

```diff
@@ -2,6 +2,7 @@ repo: https://github.com/juju-solutions/charm-containerd
 includes:
   - 'layer:basic'
   - 'layer:apt'
+  - 'layer:container-runtime-common'
```

Comment:
It's unclear from this PR why this was added, but I assume it's needed.  +1

Reply:
Yeah, it's the bit in the description that mentions "moving to container-runtime-common to gain cgroup config".  I've added that config option there because it is common across the runtimes.

Reply:
Oh, yeah.  I'm dumb.  :p